### ISSUE TITLE
Component - Accordion - Is custom widget? [OHS 2.16]

### ISSUE THREAD

**Issue by Tamayuzu** [MeD-DMC/Adapt-accessibilityfixes#145]
Accordions, when run through Accessibility Insights will report an error due to their design patterns not being recognized as accessible. There may be other elements that contain design pattern issues, we are currently looking into this.

A native widget that functions as a custom widget usually requires an ARIA widget role. The role communicates the correct function to assistive technologies and enables them to interact with the widget. 

Accessibility Insights reports that the accordion does not have the proper design pattern.

https://accessibleweb.com/question-answer/what-is-the-difference-between-native-and-custom-widgets/

What are Wai-Aria widget roles? What are all the possible ones?
https://www.w3.org/TR/wai-aria-practices-1.1/#aria_ex

Each accordion header button is wrapped in an element with role heading that has a value set for aria-level that is appropriate for the information architecture of the page.

**Comment by falcochip** [comment]
verified that widget has proper design pattern
